client/settings/upe-opt-in-banner/index.js

```diff
@@ -37,60 +49,45 @@ const Actions = styled.div`
 const ImageWrapper = styled.div`
 	background: #f7f9fc;
 	display: none;
-	flex: 0 0 33%;
+	flex: 0 0 45%;
 
 	img {
-		padding: 24px;
+		max-width: 100%;
+		width: auto;
+		height: auto;
 	}
 
 	@media ( min-width: 660px ) {
 		display: flex;
+		justify-content: center;
+		align-items: center;
 	}
 `;
 
-const UpeOptInBanner = ( props ) => (
+const UpeOptInBanner = ( { title, description, Image, ...props } ) => (
 	<BannerWrapper { ...props }>
 		<InformationWrapper>
-			<Pill>{ __( 'Early access', 'woocommerce-gateway-stripe' ) }</Pill>
-			<h3>
-				{ __(
-					'Enable the new Stripe payment management experience',
-					'woocommerce-gateway-stripe'
-				) }
-			</h3>
-			<p>
-				{ __(
-					/* eslint-disable-next-line max-len */
-					'Spend less time managing giropay and other payment methods in an improved settings and checkout experience, now available to select merchants.',
-					'woocommerce-gateway-stripe'
-				) }
-			</p>
+			<StyledPill>
+				{ __( 'Early access', 'woocommerce-gateway-stripe' ) }
+			</StyledPill>
+			<h3>{ title }</h3>
+			<p>{ description }</p>
 			<Actions>
-				<span>
-					<Button isPrimary href="?page=wc_stripe-onboarding_wizard">
-						{ __(
-							'Enable in your store',
-							'woocommerce-gateway-stripe'
-						) }
-					</Button>
-				</span>
+				<Button isSecondary href="?page=wc_stripe-onboarding_wizard">
+					{ __(
+						'Enable in your store',
+						'woocommerce-gateway-stripe'
+					) }
+				</Button>
 				<ExternalLink href="?TODO">
 					{ __( 'Learn more', 'woocommerce-gateway-stripe' ) }
 				</ExternalLink>
 			</Actions>
 		</InformationWrapper>
 		<ImageWrapper>
-			<AllPaymentMethodsIcon />
+			<Image />
 		</ImageWrapper>
 	</BannerWrapper>
 );
 
-const bannerContainer = document.getElementById(
-	'wc-stripe-upe-opt-in-banner'
-);
-
-if ( bannerContainer ) {
-	ReactDOM.render( <UpeOptInBanner />, bannerContainer );
-}
-
```

Comment:
Before: importing the banner component _also_ had side effects on the DOM - now the banner component file doesn't have side effects